tests/common/validator.go

```diff
@@ -42,7 +41,7 @@ func BuildValidator(ctx context.Context, runenv *runtime.RunEnv, initCtx *run.In
 	// Orchestrator is receiving all accounts by subscription, to then
 	// execute the `add-genesis-account` command and send back to the rest
 	// of the validators to set the initial genesis.json
-	if initCtx.GlobalSeq == 1 {
+	if seq == 1 {
```

Comment:
nit: I would change all magic numbers to custom types.

Reply:
In general, I agree that we should remove all magic numbers into something meaningful

Still, here we are just receiving back the sequence number from each of the publisher and comparing who got the first sequence, so this is self explanatory